third_party/nvidia/backend/compiler.py

```diff
@@ -51,6 +51,8 @@ def ptx_get_version(cuda_version) -> int:
     '''
     assert isinstance(cuda_version, str)
     major, minor = map(int, cuda_version.split('.'))
+    if major == 13:
```

Comment:
@xwang233 I'm confused. Isn't it already here a few lines below?

```
    if major >= 13:
        base_ptx = 90
        return base_ptx + (major - 13) * 10 + minor
```

Reply:
my bad, i only looked at the top part of the code and saw nothing above `if major == 12` 😂 

also pytorch's commit pin probably didn't have that `if major >= 13` branch yet and is failing, so i wasn't expecting that